themes/ddbasic/template.php

```diff
@@ -997,26 +996,22 @@ function ddbasic_preprocess_form_element(&$variables) {
 }
 
 /**
- * Preprocess ting_searchj_carousel.
+ * Preprocess ding_carousel.
  */
-function ddbasic_preprocess_ting_search_carousel(&$variables) {
+function ddbasic_preprocess_ding_carousel(&$variables) {
   // Add ajax to make reserve links work.
   drupal_add_library('system', 'drupal.ajax');
 
-  // The search carousel doesn't use the standard Drupal ajax API so it doesn't
+  // The ding carousel's do not use the standard Drupal ajax API so it doesn't
+  // automatically include the availability.
+  drupal_add_js(drupal_get_path('module', 'ding_availability') . '/js/ding_availability.js');
+
+  // The ding carousel's do not use the standard Drupal ajax API so it doesn't
```

Comment:
How about updating the comment instead of copy/pasting it?

```php
// The ding carousel's do not use the standard Drupal ajax API so it doesn't
// automatically include availability and covers handling.

Reply:
Updated.